cms/templates/course_outline.html

```diff
@@ -175,8 +175,11 @@ <h2 class=status-pacing-label>${_("Course Pacing:")}</h2>
                     </div>
                % endif
             </div>
-
+        % if getattr(context_course, 'language'):
```

Comment:
To avoid duplication (which can cause code to get out-of-sync if someone changes HTML in one case but forgets to change it in the other), I would change all of these if/elses so the conditional is within a single HTML tag. For example, 

````
<div class="wrapper-dnd" 
    % if getattr(context_course, 'language'):
        lang="${context_course.language}"
    % endif
>
````

I know it's not the prettiest thing to read, but I'm pretty sure it will work.